### PR TITLE
Release new snapshot

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: LATEST [hunk 1]
```diff
@@ -1,4 +1,4 @@
-1c18058f019229cd0af64669af0de31d0cec916d 1.2.0-snapshot.20200602.4310.0.1c18058f
+1386abc05ac894941f7e6eccf7e0721252957243 1.2.0-snapshot.20200603.4345.0.1386abc0
```
Comment (cocreature) [inline LATEST:1]: This should be 1.3. Keep the 1.2 line and add a new line for that.

Comment (stefanobaghino-da) [reply LATEST:1]: Do we already have a 1.2.0?

Comment (garyverhaegen-da) [reply LATEST:1]: No but we have frozen the RC, so unless someone comes up with a blocking issue, 4310 will be 1.2.

Comment (stefanobaghino-da) [reply LATEST]: Addressed by https://github.com/digital-asset/daml/pull/6207/commits/9b23c78910cf6c844baf1950c202a89525e98e4b

**Comment by stefanobaghino-da** [comment]
Manual tests passed on Linux

**Comment by stefanobaghino-da** [comment]
Manual tests passed on Windows /cc @moritzkiefer-da

**Pull request merged**